requirements.txt

```diff
@@ -0,0 +1,7 @@
+Flask==2.3.2
```

Comment:
It's important to not require a version that is too specific because it limits what the user can install for the REPL kernel.

https://github.com/ricklamers/gpt-code-ui/blob/ec7d0984bdfc8ea3b1c5eb4e54d194c181d17c1a/setup.py#L17